spec/reek/smells/too_many_instance_variables_spec.rb

```diff
@@ -58,7 +58,20 @@ def bravo
       end
     EOS
 
-    expect(src).not_to reek_of(:TooManyInstanceVariables).with_config(config)
+    expect(src).not_to reek_of(:TooManyInstanceVariables)
+  end
+
+  it 'reeks of unneeded supression' do
+    src = <<-EOS
```

Comment:
I feel that this test is not really helpful since it would succeed regardless of your code changes above.
How about rather  testing the sniff method?
Also we should test the full stack here including the output so lets add a cucmber feature for this.